src/Normalizer/Pap/AppendCampaignStatsNormalizer.php

```diff
@@ -65,6 +73,9 @@ public function normalize($object, $format = null, array $context = [])
             $campaign['nb_to_join'] = $zones ? $this->campaignHistoryRepository->countToJoinByCampaign($object, $zones) : $object->getCampaignHistoriesToJoin()->count();
             $campaign['nb_door_open'] = $zones ? $this->campaignHistoryRepository->countByCampaignAndStatus($object, CampaignHistoryStatusEnum::DOOR_OPEN, $zones) : $object->getCampaignHistoriesDoorOpen()->count();
             $campaign['nb_contact_later'] = $zones ? $this->campaignHistoryRepository->countByCampaignAndStatus($object, CampaignHistoryStatusEnum::CONTACT_LATER, $zones) : $object->getCampaignHistoriesContactLater()->count();
+            $campaign['nb_addresses_todo'] = $this->buildingStatisticsRepository->countByStatus($object, BuildingStatusEnum::TODO);
+            $campaign['nb_addresses_ongoing'] = $this->buildingStatisticsRepository->countByStatus($object, BuildingStatusEnum::ONGOING);
+            $campaign['nb_addresses_completed'] = $this->buildingStatisticsRepository->countByStatus($object, BuildingStatusEnum::COMPLETED);
```

Comment:
what about doing it in signle query: 
```suggestion
            $campaign = array_merge($campaign, $this->buildingStatisticsRepository->countByStatus($object));
```